jgrapht-core/src/main/java/org/jgrapht/alg/util/Extension.java

```diff
@@ -20,7 +20,7 @@
  * the Eclipse Foundation.
  */
 /* -----------------
- * MaximumFlowAlgorithmBase.java
```

Comment:
Got it.  Let's delete it now.  1.0.0 is already going to be a breaking release where we get rid of all old deprecated stuff, so in this case, let's skip the deprecation and just get rid of it to avoid confusion.